### PR TITLE
SNMP settings documentation

### LINKED ISSUE #276: SNMP Device Documentation Template to be completed
Complete and organize information in this template according to the associated eos template

### PR DESCRIPTION
**Pull request by xaviramon**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [x] Documentation content changes
- [ ] Other (please describe):

## Related Issue(s)

Fixes #276 
Includes documentation for  #377
## Checklist:

- [ ] I have read the [**CONTRIBUTING**](https://www.avd.sh/docs/contributing/) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed ([`pre-commit`](https://www.avd.sh/docs/installation/development/#python-virtual-environment), `make linting` and `make sanity-lint`).
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly

### CODE REVIEW

**Comment by ankudinov** [comment]
LGTM

**Review by ankudinov — APPROVED** [review]

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/snmp-settings.j2 [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{% if snmp_server is defined and snmp_server is not none %}
+### SNMP Configuration Summary
+
+| Contact | Location | Local Interface [VRF] | SNMP Traps | VRF | IPv4 ACL | IPv6 ACL |
+| ------- | -------- | --------------------- | ---------- | --- | -------- | -------- |
+| {{ snmp_server.contact | default('-') }} | {{ snmp_server.location | default('-') }} | {% if snmp_server.local_interfaces is defined and snmp_server.local_interfaces is not none %}{% for interfaces in snmp_server.local_interfaces %}- {{interfaces.name}} [{{interfaces.vrf | default('default')}}]<br>{% endfor %}{% endif %} | {% if snmp_server.traps.enable is defined and snmp_server.traps.enable is not none %}{% if snmp_server.traps.enable %} Enabled {% else %} Disabled {% endif %}{% endif %} | {% if snmp_server.vrfs is defined and snmp_server.vrfs is not none %}{% for vrf in snmp_server.vrfs %}{% if vrf.enable %}- {{vrf.name}}<br>{% endif %}{% endfor %}{% endif %} | {{ snmp_server.access_list_v4 | default('-') }} | {{ snmp_server.access_list_v6 | default('-') }} |
```
Comment (titom73) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/snmp-settings.j2]: @aphillipps. Is `<br>` a method you agree on in our documentation engine ?

Comment (ankudinov) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/snmp-settings.j2]: @titom73 @aphillipps I'd say html tags are valid. Even though avoiding them is preferred for readability, I do not see that a big problem here.

**Review by aphillipps — APPROVED** [review]
LGTM

**Pull request merged**